proto/sf/tron/type/v1/block.proto

```diff
@@ -4,103 +4,82 @@ package sf.tron.type.v1;
 option go_package = "github.com/streamingfast/firehose-tron/pb/sf/tron/type/v1;pbtron";
 
 import "google/protobuf/any.proto";
-import "api/api.proto";
 import "core/Tron.proto";
 
 // Block represents a Tron block from the RPC response
 message Block {
+  // Block hash in bytes
   bytes id = 1;
   BlockHeader header = 2;
   repeated Transaction transactions = 3;
 }
 
 // BlockHeader represents the block header from the RPC response
 message BlockHeader {
+  // Block number (height)
   uint64 number = 1;
+  // Merkle root of all transactions in the block
   bytes tx_trie_root = 2;
+  // Address of the block producer (witness)
   bytes witness_address = 3;
   uint64 parent_number = 4;
   bytes parent_hash = 5;
   uint32 version = 6;
+  // Block timestamp in milliseconds since Unix epoch
   int64 timestamp = 7;
+  // Witness signature of the block
   bytes witness_signature = 8;
 }
 
 // Transaction represents a flattened Tron transaction
 message Transaction {
-  // Basic transaction info
-  // TODO validate what field we want here
+  // Transaction hash in bytes
   bytes txid = 1;
   repeated bytes signature = 2;
   bytes ref_block_bytes = 3;
   bytes ref_block_hash = 4;
+  // Transaction expiration timestamp in milliseconds
   int64 expiration = 5;
+  // Transaction creation timestamp in milliseconds
   int64 timestamp = 6;
 
-  // TODO Do we want result and result message here?
+  // Contract execution result in bytes
+  repeated bytes constant_result = 7;
 
-  // Raw TransactionExtention
-  protocol.TransactionExtention transaction_extention = 7;
+  bool result = 8;
+  // Transaction execution status code (see response_code enum)
+  response_code code = 9;
+  // Error message if any
+  bytes message = 10;
```

Comment:
Should it be `error_message` then?

Reply:
I think the comment might be wrong here, this is from the the `Return` object (https://github.com/streamingfast/tron-protocol/blob/master/api/api.proto#L1061), not sure if it's only in case of error or not. Modified the comment but let me know what you think.